nixos/modules/services/x11/clight.nix

```diff
@@ -106,6 +106,8 @@ in {
       };
     };
 
+    systemd.user.services."app-clight@autostart".enable =
+      !lib.any (x: x == "/etc/xdg/autostart") config.environment.pathsToLink;
```

Comment:
Why don't we use xdg.autostart.enable? It would probably be a lot faster. Also since it is enabled by default, why do we even have an alternative Implementation in the first place?